manifests/stage-ofed-driver/0100_service_account.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ofed-driver
+  namespace: mlnx-network-operator
```

Comment:
as an FYI, the resources network-operator deploys are now in a fixed namespace : `mlnx-network-operator-resources`
does this service account need be deployed in that namespace ? if so, you should use the template : `namespace: {{ .RuntimeSpec.Namespace }}`

Reply:
Ack changed the namespaces for the `manifests/stage-ofed-driver/*.yaml`